app/assets/javascripts/miq_explorer.js

```diff
@@ -0,0 +1,167 @@
+ManageIQ.explorer = {}
```

Comment:
Could you move this one to `miq_global.js` please? Just so that we have all the fields mentioned (and commented) in one place...